docs/UserGuide.md

```diff
@@ -112,7 +112,7 @@ After entering this command, you should see a pop-up window similar to the one s
 
 ## Viewing expired food items
 
-When you launch the app, it checks for any expired food present in your food inventory.
+When you launch the app, it checks for any expired food present in your food inventory. This lets you easily see what foods have expired.
```

Comment:
Maybe can change "food" to "food item" to make it consistent with the rest of the UG? Same for "foods" as well.

Reply:
Updated, thanks.